packages/fields/src/types/CloudinaryImage/index.js

```diff
@@ -19,5 +20,11 @@ export default {
     mongoose: MongoCloudinaryImageInterface,
     knex: KnexCloudinaryImageInterface,
   },
-  block: CloudinaryBlock,
+  blocks: {
+    image: {
+      type: 'cloudinaryImage',
+      viewPath: imageContainer.viewPath,
```

Comment:
Maybe instead of viewPath, we should have a views key for consistency with field types and in case we want to add more views in the future?

Reply:
Yes! It's a great idea. I went back and forth on this  one, especially since a block really can have multiple views: The Toolbar, the block itself, the pop-up thingy on the left, etc.

But then when it came down to it; they're all exported by the one view. And I can't really think of a case where they'd need to be separately loaded (with the `viewPath`, it can already `import` other files if it needs).

Given `viewPath` is how all the blocks currently do it ([for example](packages/fields/src/types/Content/blocks/blockquote.js)), I think it best if we revisit this in the future if we need to change the API at all.